dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -156,7 +150,11 @@ def install(self):
                 raise ImportError(error_msg)
 
         print("Proceed with build...")
-        dali_cflags, dali_lflags = get_dali_build_flags()
+        if self.dali_lib_path == "":
+            dali_lflags = ""
+            dali_cflags = "-I" + os.path.join(self.src_path, "include")
+        else:
```

Comment:
Do we need this else here?

Reply:
My point was, if we already have DALI installed, why not linking with it. I wasn't sure about it though. Do you think I should skip linking always?

Reply:
I just wonder if this can lead to some strange problems when you link with the headers of DALI you have not the one you bundle (especially during the development or something).

Reply:
Makes sense, let's limit to one execution path